src/input.styles.ts

```diff
@@ -105,12 +113,12 @@ export default [
       /* The input obscures an offset outline for -webkit-calendar-picker-indicator, so 'focus-outline' is not used */
       &[type='date'] {
         &::-webkit-calendar-picker-indicator {
-          border-radius: var(--glide-core-border-radius-xs);
-          padding: var(--glide-core-spacing-xxs);
+          border-radius: 0.125rem;
```

Comment:
Ah. So we will still have some hardcoded values.

Reply:
Yeah, unfortunately in a few cases.  It's on my list to go back through cases like this one and see if we can remove it in favor of another variable though!

Reply:
Totally agree!